src/runtime-rs/crates/hypervisor/src/remote/mod.rs

```diff
@@ -43,8 +44,14 @@ impl Remote {
 
 #[async_trait]
 impl Hypervisor for Remote {
-    async fn prepare_vm(&self, id: &str, netns: Option<String>) -> Result<()> {
+    async fn prepare_vm(
+        &self,
+        id: &str,
+        netns: Option<String>,
+        annotations: &HashMap<String, String>,
+    ) -> Result<()> {
         let mut inner = self.inner.write().await;
+        inner.set_annotations(annotations);
```

Comment:
Frankly, as a matter of style I'd prefer if you pass `annotations` to `RemoteInner::prepare_vm()` and keep this function's implementation trivial as is the convention.  This works too though...